### PR TITLE
Issue 13768 - Privacy and Security risk - Downloads are logged in private tab

### PR DESCRIPTION
**Pull request by a-one-d-one**
Disclaimer - Work In Progress

Hi there I am not sure if this is the correct protocol to dealing with issues on brave. I have been able to partially implement this but need some help in getting it completely fixed. 

As of now the downloads are removed from the "Downloads Bar" but still persist in the URL "about:downloads" I have attached the following to illustrate :   

apr-21-2018 18-18-07

I would appreciate some help and guidance on how to proceed in order to get it removed from the URL as well.

### CODE REVIEW

**Comment by bsclifton** [comment]
NOTE: this PR is meant to fix https://github.com/brave/browser-laptop/issues/13768

@a-one-d-one you might start by looking at the code for the about:downloads page
https://github.com/brave/browser-laptop/blob/master/js/about/downloads.js

The page receives an event with the data that it shows:
https://github.com/brave/browser-laptop/blob/7cc0aeffec724fcae18046c2000ca5aa222fe2b5/js/about/downloads.js#L67-L72

If you search for that event name, you'll find that it's being sent from the tabs object:
https://github.com/brave/browser-laptop/blob/7cc0aeffec724fcae18046c2000ca5aa222fe2b5/app/browser/tabs.js#L304-L308

Basically, all entries in the `downloads` array in appState are being shown. You're going to want to find where `downloads` gets set / appended to.  I searched for `.set('downloads'` and found that there's a few places that are setting it in the downloadsReducer:
https://github.com/brave/browser-laptop/blob/master/app/browser/reducers/downloadsReducer.js

**Comment by bsclifton** [comment]
@a-one-d-one any update on this? Let us know if you need help 😄

**Comment by bsclifton** [comment]
Closing as stale
